src/jit/lir.h

```diff
@@ -123,39 +141,31 @@ class LIR final
         bool IsSubRange() const;
 
         GenTree* FirstNonPhiNode() const;
+        GenTree* FirstNonPhiOrCatchArgNode() const;
 
```

Comment:
Do we need both of these? I wonder if it should be just "FirstSignificantNode()" and we can define "significant" however we need.